comment-updater-core/src/main/kotlin/org/jetbrains/research/commentupdater/models/config/ModelFilesConfig.kt

```diff
@@ -0,0 +1,11 @@
+package org.jetbrains.research.commentupdater.models.config
+
+import java.io.File
+
+class ModelFilesConfig(dataDir: File = File("/Users/Ivan.Pavlov/IdeaProjects/CommentUpdater1")) {
```

Comment:
I thought we agreed that the user should pass the path as an argument to the plugin